### PR TITLE
vitasdk-update: Add a script to update an existing vitasdk installation

### PR DESCRIPTION
**Pull request by xyzz**
This is to be installed into vitasdk by buildscripts.

The installation flow remains compatible (bootstrap-vitasdk, then install-all)
but now the user can run vitasdk-update to update the SDK and packages.

### CODE REVIEW

File: include/install-packages.sh [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+set -e
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+b() {
+	$DIR/../vdpm $1
+}
+
+install_packages() {
```
Comment (devnoname120) [inline include/install-packages.sh:11]: Maintaining two identical lists is annoying, maybe do a `git checkout` on https://github.com/vitasdk/packages/blob/master/travis_packages.sh?

File: include/install-vitasdk.sh [hunk 2]
```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+
+get_download_link () {
+  curl "https://api.github.com/repos/vitasdk/autobuilds/releases" | grep "master" | grep "browser_download_url" | grep $1 | head -n 1 | cut -d '"' -f 4
```
Comment (devnoname120) [inline include/install-vitasdk.sh:4]: GitHub API's keeps being rate-limited from Travis, use this instead: https://github.com/vitasdk/vita-headers/blob/master/.travis.d/download_sdk.sh

File: vitasdk-update [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+
+if [ -z "$VITASDK" ]; then
+  echo '$VITASDK is not set'
+  exit 1
+fi
+
+. $DIR/include/install-vitasdk.sh
```
Comment (devnoname120) [inline vitasdk-update:13]: Note that renamed headers and stubs will be duplicated when this script is executed.

File: include/install-packages.sh (hunk 1 above)
Comment (xyzz) [reply include/install-packages.sh:11]: We can install this in buildscripts, but then it won't work out of the `vdpm` checkout (I'm not sure what you mean by do a `git checkout` of it)
also, given https://github.com/vitasdk/packages/pull/28#issuecomment-309392506 we might want to build some stuff twice eventually, this shouldn't matter much now but I'm thinking how much the two lists will diverge eventually

File: include/install-vitasdk.sh (hunk 2 above)
Comment (xyzz) [reply include/install-vitasdk.sh:4]: this would add a dep on python

File: include/install-packages.sh (hunk 1 above)
Comment (devnoname120) [reply include/install-packages.sh:11]: I would suggest to do 
`git archive --remote=ssh://git@github.com/vitasdk/packages.git HEAD travis_packages.sh | tar -x` but apparently GitHub doesn't support `git-archive`.

Maybe source `https://raw.githubusercontent.com/vitasdk/packages/master/travis_packages.sh`?

**Pull request merged**